roles/repos/defaults/main.yml

```diff
@@ -0,0 +1 @@
+mantl_version: 1.1.0
```

Comment:
Yes. I'm using a GUI to push git changes, and apparently there were conflicts and my changes were not pushed to the remote branch... I'll sort it out, thanks for a notice.